src/dotnet-svcutil/lib/src/Shared/TargetFrameworkHelper.cs

```diff
@@ -63,6 +63,7 @@ internal class TargetFrameworkHelper
                 ProjectDependency.FromPackage("System.ServiceModel.Http", "4.8.*"    ),
                 ProjectDependency.FromPackage("System.ServiceModel.NetTcp", "4.8.*"  ),
                 ProjectDependency.FromPackage("System.ServiceModel.Security", "4.8.*"),
+                ProjectDependency.FromPackage("System.ServiceModel.Federation", "4.8.*"),
             } }
```

Comment:
We don't support Federation with svcutil yet so no need to add that package.

Reply:
I have removed that package